cirq/ops/parallel_gate_operation.py

```diff
@@ -87,7 +87,7 @@ def _apply_unitary_(self, args: 'protocols.ApplyUnitaryArgs'
         """
         if not protocols.has_unitary(self.gate):
             return NotImplemented
-        return protocols.apply_unitaries((self.gate(q) for q in self.qubits),
+        return protocols.apply_unitaries((self._gate(q) for q in self.qubits),
```

Comment:
Why change this?  The rest of the class still uses `self.gate`?

Reply:
I was getting mypy errors here, but can't reproduce them now. Reverted.

Reply:
Oh right, it's pylint that is not happy because it sees that `self.gate` is a property and doesn't like that we're calling it. Fixed by using `self.gate.on(q)`.